### PR TITLE
Fix table name

### PR DESCRIPTION
**Pull request by gnovaro**
user_token -> users_tokens

### CODE REVIEW

File: auth-schema-mysql.sql [hunk 1]
```diff
@@ -18,7 +18,7 @@ CREATE TABLE IF NOT EXISTS `roles_users` (
 
 CREATE TABLE IF NOT EXISTS `users` (
   `id` int(11) UNSIGNED NOT NULL AUTO_INCREMENT,
-  `email` varchar(254) NOT NULL,
+  `email` varchar(127) NOT NULL,
```
Comment (zeelot) [inline auth-schema-mysql.sql:21]: Why did you make this change?

Comment (gnovaro) [inline auth-schema-mysql.sql:21]: I had a wrong wrong information about email longitude

Comment (kemo) [inline auth-schema-mysql.sql:21]: @gnovaro can you remove this change in your branch? (it should be added to the PR, just commit & push)

**Comment by kemo** [comment]
I don't see a reason why this table should be called users_tokens if the User_Token model refers to user_tokens already.

Let me know if there is a valid reason to change it and I will reopen this. Closing

**Pull request closed without merging**